app/Http/Controllers/Controllers/GatwickController.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace App\Http\Controllers\Controllers;
+
+use App\Http\Controllers\Controller;
+use Carbon\Carbon;
+use Redirect;
+
+class GatwickController extends \App\Http\Controllers\BaseController
+{
+    public function getIndex()
+    {
+        $account = $this->account;
+
+        $groupone = $this->account->networkDataAtc()
+            ->whereBetween('connected_at', [Carbon::now()->subMonth(3), Carbon::now()])
+            ->where(function ($callsigns) {
+                $callsigns->where('callsign', 'like', 'EGCC_%')
```

Comment:
Could this also be refactored into a scope? 
  public static function scopeWithCallsign($query, $callsign)
    {
        return $query->where('callsign', 'LIKE', $callsign);
    }

we could make a new one of these to accept an array of callsigns, somthing like
public static function scopeWithManyCallsigns($query, array $callsigns)
    {
        $query->where('callsign', 'LIKE', $callsigns[0]);
        $callsigns = array_shift($callsigns)
        foreach($callsigns as $callsign){
            $query->orWhere('callsign', 'LIKE', $callsign);
         }
        return $query;
    }